### PR TITLE
Fixes passing wrong argument to createWithTTF

### PR DESCRIPTION
**Pull request by dumganhar**
Re: cocos-creator/fireball#

Changes proposed in this pull request:
 * 

> Makes sure these boxes are checked before submitting your PR - thank you!
>
- [X] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
- [X] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
- To official teams:
  - [X] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
  - [X] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
  - [X] Make sure any runtime log information in `cc.log` , `cc.error`, `cc.warn` or `cc.assert` has been moved into `DebugInfos.js` with an ID

@cocos-creator/engine-admins

### CODE REVIEW

**Comment by knoxHuang** [comment]
LGTM

**Comment by jareguo** [comment]
changelog 该怎么说？麻烦描述下

**Review by jareguo — APPROVED** [review]

File: jsb/jsb-label.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ cc.Label = function (string, fontHandle, spriteFrame, fontSize) {
             customGlyphs: "",
             distanceFieldEnable: false
         };
-        label = jsbLabel.createWithTTF(ttfConfig, string, this._fontSize);
+        label = jsbLabel.createWithTTF(ttfConfig, string);
```
Comment (dumganhar) [inline jsb/jsb-label.js:267]: If we pass the 3rd argument, it should be `hAlignment` rather than fontSize.
Font size is defined in ttfConfig.

**Comment by dumganhar** [comment]
修复RichText使用自定义TTF字体的崩溃问题

**Pull request merged**